testsuite/tests/grafana/test_dashboards.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for Grafana Dashboards definitions"""
+import pytest
+
+from testsuite.capabilities import Capability
+
+pytestmark = [
+        pytest.mark.skipif("TESTED_VERSION < Version('2.9')"),
+        pytest.mark.required_capabilities(Capability.OCP4),
+    ]
+
+# pylint: disable=line-too-long
+DASHBOARDS = [
+    ("apicast-mainapp",
+     "sum(rate(nginx_http_connections{namespace='$namespace', pod=~'apicast-$env.*'}[1m])) by (state)"),
+    ("apicast-services",
+     "sum(rate(upstream_status{namespace='$namespace', pod=~'apicast-$env.*', service_id='$service_id'}[1m])) by (status)"),  # noqa: E501
```

Comment:
Not sure if it won't degrade the readability, but if you want to remove the long lines you can use this method of splitting it https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation.

Reply:
I split the lines.